superset/db_engine_specs/presto.py

```diff
@@ -373,6 +377,55 @@ def select_star(
             presto_cols,
         )
 
+    @classmethod
+    def estimate_statement_cost(cls, statement, database, cursor, user_name):
+        db_engine_spec = database.db_engine_spec
+        parsed_query = ParsedQuery(statement)
+        sql = parsed_query.stripped()
+
+        SQL_QUERY_MUTATOR = config.get("SQL_QUERY_MUTATOR")
+        if SQL_QUERY_MUTATOR:
+            sql = SQL_QUERY_MUTATOR(sql, user_name, security_manager, database)
+
+        sql = f"EXPLAIN (TYPE IO, FORMAT JSON) {sql}"
+
+        db_engine_spec.execute(cursor, sql)
+        polled = cursor.poll()
+        while polled:
+            time.sleep(0.2)
+            polled = cursor.poll()
+        data = db_engine_spec.fetch_data(cursor, 1)
```

Comment:
can you add a comment showing what this response looks like? It's not super obvious from the code how the result is being pulled out

Reply:
Will do!